src/main/scala/cromwell/util/WriteOnceStore.scala

```diff
@@ -21,4 +21,21 @@ class WriteOnceStore[K, V] {
    * @return An immutable copy of the current store map
    */
   def toMap: Map[K, V] = writeOnceStore.readOnlySnapshot().toMap
-}
\ No newline at end of file
+
+  /**
+    * Remove an entry in the store as specified by key.
+    * @param key Key of the entry to remove.
+    * @return `Some(value)` if the entry is present in the store, otherwise `None`.
+    */
+  def remove(key: K): Option[V] = writeOnceStore.remove(key)
```

Comment:
I actually just saw a TODO higher up this PR saying "... and this lets us remove the WriteOnceStore!" so hopefully this is about to disappear?